### PR TITLE
Add support for Sitecore 9.1.1 xConnect indexes

### PR DESCRIPTION
**Pull request by vtml**
A really quick and dirty way to add support for Sitecore 9.1.1 xConnect indexes
https://github.com/searchstax/searchstax-sitecore-plugin/issues/2
https://github.com/searchstax/searchstax-sitecore-plugin/issues/9

### CODE REVIEW

File: xconnect-searchstax-connector-9.1.1.ps1 [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    Write-Host $solr
+    if ($solrUsername.length -gt 0){
+        $secpasswd = ConvertTo-SecureString $solrPassword -AsPlainText -Force
+        $credential = New-Object System.Management.Automation.PSCredential($solrUsername, $secpasswd)
+    }
+    "Creating Collections ... "
+    foreach($collection in $collections){
+        $collection | Write-Host
+        if ($solrVersion -eq "7.2.1") {
+            $url = -join($solr, "admin/collections?action=CREATE&name=",$sitecorePrefix,$collection,"&numShards=1&replicationFactor=",$nodeCount,"&collection.configName=sitecore-xdb1_$sitecorePrefix")
```
Comment (klpatil) [inline xconnect-searchstax-connector-9.1.1.ps1]: this needs to be renamed from "sitecore-xdb1_" to "sitecore-xdb" - Which should match with original uploaded config file

Comment (klpatil) [reply xconnect-searchstax-connector-9.1.1.ps1]: Rest all is good

**Review by klpatil — COMMENTED** [review]
Rest all is good

File: xconnect-searchstax-connector-9.1.1.ps1 (hunk 1 above)
Comment (vtml) [reply xconnect-searchstax-connector-9.1.1.ps1]: This is now done.

**Pull request closed without merging**